public/wp-config.php-dist

```diff
@@ -92,3 +92,9 @@ if ( !defined('ABSPATH') )
 
 /** Sets up WordPress vars and included files. */
 require_once(ABSPATH . 'wp-settings.php');
+
+/* ChapPress-For logging purposes only, not to be used for live site */
+$messageLog  = '';
+$messageLog .= "[" . date('d-M-Y H:i:s e') . "]" . " - "; 
+$messageLog .= "Hello from the config file \n"; 
+error_log($messageLog, 3, "./wp-content/debug.log");

```

Comment:
It worked. Nice. I'd prefer a simpler interface but that can be address in the future.